### PR TITLE
Test label

### PR DESCRIPTION
**Pull request by scopatz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

I found a bit of time to sneak this out

CC @isuruf @jakirkham

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge-admin please rerender

**Comment by isuruf** [comment]
See https://github.com/conda-forge/conda-smithy/issues/897

**Comment by jakirkham** [comment]
Yeah we just need to move this to `conda_build_config.yaml`. Here's [an example]( https://github.com/conda-forge/ucx-split-feedstock/blob/d0770c1f66a482feb9ff1ec53bbe8bcb9f7e8320/recipe/conda_build_config.yaml ).

**Comment by scopatz** [comment]
Are the conda-forge docs out of date then?

**Comment by scopatz** [comment]
updated!

**Comment by isuruf** [comment]
> Are the conda-forge docs out of date then?

Yes. To paraphrase you from that issue,

> Improved docs would be great to have!

:smile:

**Comment by scopatz** [comment]
Haha yeah, I still agree with myself :wink:

File: recipe/build.py [hunk 1]
```diff
@@ -352,12 +352,13 @@ def extract(self):
...
                 cmd = [
                     os.path.join(self.src_dir, self.runfile),
-                    "--installpath=%s" % (tmpd),
+                    f"--installpath={tmpd}",
                     "--toolkit",
                     "--silent",
                     "--override",
                     "--nox11",
                 ]
+                print(f"Extract command: {' '.join(cmd)}")
```
Comment (kkraus14) [inline recipe/build.py:361]: Do we want to leave this print in here?

File: conda-forge.yml [hunk 2]
```diff
@@ -1 +1,4 @@
 conda_forge_output_validation: true
+channels:
+  targets:
+    - ["conda-forge", "test"]
```
Comment (jakirkham) [inline conda-forge.yml:4]: This can now be dropped 🙂

```suggestion
```

File: recipe/build.py (hunk 1 above)
Comment (scopatz) [reply recipe/build.py:361]: It doesn't hurt. It helps with debugging

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

File: recipe/build.py (hunk 1 above)
Comment (jakirkham) [reply recipe/build.py:361]: Yeah I guess we don't see what command was called if we remove this right? Seems worth keeping in that case.

**Review by jakirkham — APPROVED** [review]

**Review by kkraus14 — APPROVED** [review]

**Comment by jakirkham** [comment]
@isuruf, are you happy with this or are there more changes needed?

**Comment by jakirkham** [comment]
Going to go ahead and merge so this doesn't block other PRs in the interim. If there are other things that need to be addressed, please let us know and we can follow-up. Thanks!

**Comment by jakirkham** [comment]
Thanks Anthony and everyone for reviews! 😄

**Comment by scopatz** [comment]
Thanks All!

**Pull request merged**